src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -94,13 +100,33 @@ private static List<string> GetFingerPrint(LogEventInfo logEvent)
             return fingerPrint;
         }
 
+        private bool IsSentryMessage(LogEventInfo logEvent)
+        {
+            if (logEvent.Properties.ContainsKey("Sentry"))
+            {
+                return logEvent.Properties["Sentry"] != null;
+            }
+
+            if (logEvent.Level >= LogLevel.Error && logEvent.Exception != null)
+            {
+                return true;
+            }
+
+            return false;
+        }
+
 
         protected override void Write(LogEventInfo logEvent)
         {
+            if (_unauthorized)
```

Comment:
Would it be worth changing this to a variable holding authorized instead? Unauthorized seems like it could be a bit confusing in a conditional.

Reply:
Unauthorized refers to the HTTP 401 status code which shares the same name. I think it makes since to keep it in this case.